packages/flutter/lib/src/material/chip.dart

```diff
@@ -2064,19 +2064,25 @@ class _RenderChip extends RenderBox with SlottedContainerRenderObjectMixin<_Chip
     }
   }
 
+  final LayerHandle<OpacityLayer> _childOpacityLayerHandler = LayerHandle<OpacityLayer>();
+
   void _paintChild(PaintingContext context, Offset offset, RenderBox? child, bool? isEnabled) {
+    final OpacityLayer? oldLayer = _childOpacityLayerHandler.layer;
+    _childOpacityLayerHandler.layer = null;
```

Comment:
My comment in https://github.com/flutter/flutter/pull/134708#discussion_r1333333357 applies here as well.